src/drivers/net/ath/ath5k/ath5k_eeprom.c

```diff
@@ -410,7 +410,7 @@ ath5k_eeprom_read_turbo_modes(struct ath5k_hw *ah,
 {
 	struct ath5k_eeprom_info *ee = &ah->ah_capabilities.cap_eeprom;
 	u32 o = *offset;
-	u16 val;
+	u16 val = 0;
```

Comment:
Should be fixed more generally in https://github.com/ipxe/ipxe/commit/065dce8d5

Thanks for pointing this one out!